### ISSUE TITLE
Youtube feeds problem in macOS

### ISSUE THREAD

**Issue by LitoFrame** [Ranchero-Software/NetNewsWire#3308]
I have an issue with my youtube feeds on macOS on iCloud account. They are not updating and if I delete one of them I can't add it back. It said "Feed not found".
I can add the feed in iOS and then it appear in my mac. Same with the update, when I open NNW on my iPhone the feeds updating on my mac as well.
I use NNW Version 6.0.3 (6035) on macOS 11.6

**Comment by babbage** [comment]
Could you provide a link to one of the feeds that you are seeing this issue with, @LitoFrame? It’s a good starting point for testing if we’re looking at the same feed…

**Comment by LitoFrame** [comment]
https://www.youtube.com/feeds/videos.xml?channel_id=UCGWkDcYbDKP9r--ym28YwAQ
Of course now it is working. :) I could re-add all my youtube channels. 

In the meantime I deleted ~/Library/Containers/com.ranchero.NetNewsWire-Evergreen/Data/Library/Application Support/NetNewsWire/Accounts/2_iCloud/Sync.sqlite3
It solved sync issue in the past for example when there was a power cut durning NNW open. After restart my computer NNW didn't sync at all. I'm not sure it helped now...

Can this trouble cause VPN or firewall? I found this problem maybe in 2 days ago. I just came into my mind first when I tried to re-add my channel I can't open the feed in a browser, but now I can open all. So maybe the problem is in my side.
Next time if I found it again I will turn off VPN...
